reVX/utilities/reeds_cols.py

```diff
@@ -127,10 +133,12 @@ def add_extra_data(data_frame, extra_data, merge_col="sc_point_gid"):
         dsets = data_info["dsets"]
         data_fp = data_info["data_fp"]
 
-        if data_fp.endswith(".json"):
+        if isinstance(data_fp, dict):
+            extra_data = data_fp
+        elif str(data_fp).endswith(".json"):
```

Comment:
Is this cast to `str` (and the one below) really necessary? This value should either be a string or a dictionary, and we already check for a dictionary above. Any other kind of input *should* error out here.

Reply:
It could be a Posix Path, can add to the docs.

Reply:
Ah gotcha. I don't think you need to update the docs then, unless you really want to